web_switch_context_warning/static/src/js/switch_context_warning_worker.js

```diff
@@ -0,0 +1,26 @@
+// Show a big banner in the top of the page if the company has been
+// changed in another tab or window (in the same browser)
+var con = [];
+var lastCtx = null;
+
+addEventListener("connect", function (ee) {
```

Comment:
Who fires this `connect` event? This does not get executed for me (in a vanilla Odoo instance without anything else installed). That stops me from testing further.